contrib/babelfishpg_tds/src/backend/tds/tdsprotocol.c

```diff
@@ -77,6 +79,45 @@ static void ResetTDSConnection(void);
 static TDSRequest GetTDSRequest(bool *resetProtocol);
 static void ProcessTDSRequest(TDSRequest request);
 
+
+static void attention_listener_handler(SIGNAL_ARGS)
```

Comment:
Somewhere we should have a comment describing why this is even needed (either here or where we use the signal handler... or even both).